core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -278,7 +198,7 @@ private[ui] class StagePage(parent: StagesTab, store: AppStatusStore) extends We
       }
     }
     val currentTime = System.currentTimeMillis()
-    val (taskTable, taskTableHTML) = try {
+    val taskTable = try {
       val _taskTable = new TaskPagedTable(
```

Comment:
seems like this shouldn't be needed anymore.  Feels like a bunch of stuff in here should be able to be removed now?

Reply:
ok might be something we should look at closer to see if we want to clean up and use the rest api for